src/coreclr/vm/eventpipeadapter.h

```diff
@@ -318,7 +318,7 @@ class EventPipeAdapter final
 		ep_add_provider_to_session (provider, session);
 	}
 
-	static inline EventPipeProvider * CreateProvider(const SString &providerName, EventPipeCallback callback, void* callbackContext = nullptr)
+	static inline EventPipeProvider * CreateProvider(const WCHAR* providerName, EventPipeCallback callback, void* callbackContext = nullptr)
```

Comment:
This also gets used by the generated code for event pipe in coreclr:
https://github.com/dotnet/runtime/blob/a6cf8d680845727194eaef9636253f6115f2985f/src/coreclr/scripts/genEventPipe.py#L157

You probably want to make it stop using `SL`:
https://github.com/dotnet/runtime/blob/a6cf8d680845727194eaef9636253f6115f2985f/src/coreclr/scripts/genEventPipe.py#L181-L183